.github/workflows/CI.yml

```diff
@@ -64,13 +59,11 @@ jobs:
         with:
           command: fmt
           args: --all -- --check
-      - run: make regen_check
```

Comment:
Why remove this one?

Reply:
We don't use the make script any more and the generator doesn't have a assume yes flag yet. Will have to fix that first, same with gtk-rs repository

Reply:
Ok, please create an issue for that and then merge this here :)

Reply:
Done https://github.com/gtk-rs/gtk4/issues/39